tests/castle-vault.ts

```diff
@@ -1159,37 +1210,48 @@ describe("castle-vault", () => {
                 await port.getUnclaimedStakingRewards(program);
             assert.equal(rewardAmountAfterClaiming, 0);
 
-            const mint = port.accounts.stakingRewardTokenMint;
-            const rewardToken = new Token(
-                program.provider.connection,
-                mint,
-                TOKEN_PROGRAM_ID,
-                Keypair.generate()
+            const claimedRewardAmount = await getSplTokenAccountBalance(
+                port.accounts.stakingRewardTokenMint,
+                port.accounts.vaultPortRewardToken
             );
-            const claimedRewardAmount = (
-                await rewardToken.getAccountInfo(
-                    port.accounts.vaultPortRewardToken
-                )
-            ).amount.toNumber();
             assert.isAtLeast(claimedRewardAmount, accumulatedRewardAmount);
 
             if (subReward) {
-                const subRewardMint = port.accounts.stakingSubRewardTokenMint;
-                const subRewardToken = new Token(
-                    program.provider.connection,
-                    subRewardMint,
-                    TOKEN_PROGRAM_ID,
-                    Keypair.generate()
+                const claimedSubRewardAmount = await getSplTokenAccountBalance(
+                    port.accounts.stakingSubRewardTokenMint,
+                    port.accounts.vaultPortSubRewardToken
                 );
-                const claimedSubRewardAmount = (
-                    await subRewardToken.getAccountInfo(
-                        port.accounts.vaultPortSubRewardToken
-                    )
-                ).amount.toNumber();
+                // const claimedSubRewardAmount = (
```

Comment:
Can you comment why this is commented out, or delete?

Reply:
It's a mistake, I'll correct it.